tools/docker/tests/flows/__init__.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python3
```

Comment:
For the hashbang line: OK, good point
For the tests part: I've actually already done this, just not pushed the commit yet, because I'm still fixing some of the stuff that breaks and I don't feel comfortable pushing commits that break functionality. So it will be part of the merge eventually, but I may just split the work into multiple parts, of which the move of the tests to a top level directory would be one of those.